README.md

```diff
@@ -42,7 +42,7 @@ The simplest starting point is to set the global tracer. As early as possible, d
 
 ### Creating a Span given an existing Request
 
-To start a new `Span`, you can use the `StartSpanFromContext` method.
+To start a new `Span`, you can use the `startActiveSpan` method.
```

Comment:
You name it `startActiveSpan` here and then the example uses `startManualSpan` though. Is this right?